frontend/src/theme/foundations/colors.js

```diff
@@ -1,7 +1,7 @@
 export default {
   brand: {
     royal: "#724A96",
-    purple: "#CC99FF",
+    purple: "#724A96",
```

Comment:
This change might impact a lot of places, is that fine?

Reply:
Whoops that was a mistake. I reverted it.